doc/visual-programming/source/widgets/visualize/modelmapprojectionrank.rst

```diff
@@ -1,71 +0,0 @@
-ModelMap Projection Rank
-========================
-
-.. figure:: icons/modelmap-projection.png
```

Comment:
Images used only by this file should also be removed. If possible, removal of file and corresponding images should be moved to a separate commit.